src/routes/blog/post/announcing-messaging-set-up-a-messaging-service

```diff
@@ -0,0 +1,73 @@
+
+## All your messages in one place
+
+As mentioned, Messaging covers every communication avenue under one unified API, allowing you to send email, SMS, and push notifications from your Appwrite project. It connects with a variety of third-party providers to deliver you messages. See this table to get an overview of what message types you can send with each provider.
+
+| Message Type      | Provider                                         |
+|-------------------|--------------------------------------------------|
+| Push Notifications| - APNs (Apple)                                   |
+|                   | - FCM (Google)                                   |
+| Email             | - Mailgun                                        |
+|                   | - Sendgrid                                       |
+|                   | - SMTP (Generic)                                 |
+| SMS               | - Twilio                                         |
+|                   | - MSG91                                          |
+|                   | - Telesign                                       |
+|                   | - TextMagic                                      |
+|                   | - Vonage                                         |
+
+
+## Message your existing users
+
+With Messaging, you can deliver messages to groups of users through topics or address individual users and their devices through targets. Messaging deeply integrates with integrated existing accounts in Appwrite out of the box. You can immediately start messaging your existing users and integrate deeply with other Appwrite features.
+
+You can address recipients of messages at three levels:
+
+- Topics, which hold a group of users’ devices.
+- Users which holds all targets associated with their account.
+- Targets, which are individual emails, phone numbers, and devices.
+
+You can publish announcements and newsletters through topics, deliver user-specific promotions and alerts to all users’ devices, and address individual targets for location-based or real-time alerts.
+
+![Creating a message from the Console](/images/blog/message-overview.png)
+
+## Deliver messages your way
+
+Draft, schedule, and send your messages in the Appwrite Console, or use Appwrite SDKs to send messages through code.
+
+Drafting directly in the Appwrite Console is great for promo messages, newsletters, and other announcements. You can see a preview of what your message will look like right in the console and control how your message will look before sending.
+
+For real-time alerts, you can programmatically send messages to your users and integrate with Appwrite Functions triggered by platform events. This allows you to deliver automated messages like invoices, login alerts, order confirmations, and new message alerts.
+
+## Get started with Messaging
+
+To get started with messaging, you connect a provider, add topics and targets, and send your first message today for your project.
```

Comment:
A paragraph about many more adapters to come and call to the community to contribute more could be helpful here.